conftest.py

```diff
@@ -181,42 +158,51 @@ def forest():
         return pytest.skip("This test requires a Forest connection: {}".format(e))
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope="session")
 def benchmarker():
     try:
         bm = get_benchmarker(timeout=2)
         bm.apply_clifford_to_pauli(Program(I(0)), sX(0))
         return bm
     except (RequestException, TimeoutError) as e:
-        return pytest.skip("This test requires a running local benchmarker endpoint (ie quilc): {}"
-                           .format(e))
+        return pytest.skip(
+            "This test requires a running local benchmarker endpoint (ie quilc): {}".format(e)
+        )
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope="session")
 def local_qvm_quilc():
     """Execute test with local qvm and quilc running"""
-    if shutil.which('qvm') is None or shutil.which('quilc') is None:
-        return pytest.skip("This test requires 'qvm' and 'quilc' "
-                           "executables to be installed locally.")
+    if shutil.which("qvm") is None or shutil.which("quilc") is None:
+        return pytest.skip(
+            "This test requires 'qvm' and 'quilc' " "executables to be installed locally."
```

Comment:
weird

Reply:
lol

Reply:
Called out in in the black docs as something they intend to fix, so we can consider this more of a temporary artifact than a conscious style decision. Issue tracked here: https://github.com/psf/black/issues/26

Reply:
I went ahead and fixed them all anyways!